components/eamxx/src/physics/shoc/eamxx_shoc_process_interface.cpp

```diff
@@ -432,6 +432,10 @@ void SHOCMacrophysics::run_impl (const double dt)
                        shoc_preprocess);
   Kokkos::fence();
 
+  if (m_params.get<bool>("check_flux_state_consistency", false)) {
```

Comment:
Should this be called always? Or toggled on and off via input.yaml?

Reply:
i would keep on/off switch if you already implemented it.